xbmc/addons/PVRClient.h

```diff
@@ -707,6 +708,13 @@ namespace PVR
     static void WriteClientChannelInfo(const CPVRChannelPtr &xbmcChannel, PVR_CHANNEL &addonChannel);
 
     /*!
+     * @brief Copy over epg info from CEpgInfoTag to EPG_TAG.
+     * @param tag The epg tag on XBMC's side.
+     * @param addonChannel The epg tag on the addon's side.
```

Comment:
There is no param addonChannel. please fix.

And please, don't mis(use) me as your doxy / typo checker. That's no fun. Do this by yourself, please, before committing. Thanks.